Firestore/Firestore.podspec

```diff
@@ -51,6 +51,7 @@ Google Cloud Firestore is a NoSQL document database built for automatic scaling,
 
   s.pod_target_xcconfig = {
     'GCC_PREPROCESSOR_DEFINITIONS' => 'GPB_USE_PROTOBUF_FRAMEWORK_IMPORTS=1 ',
+    'HEADER_SEARCH_PATHS' => '"${PODS_ROOT}" "${PODS_ROOT}/../../.."',
```

Comment:
Use PODS_TARGET_SRCROOT instead of PODS_ROOT - see https://github.com/CocoaPods/CocoaPods/issues/7213#issuecomment-343305527

Reply:
Done.